src/lib/components/input/input-mixins.scss

```diff
@@ -0,0 +1,52 @@
+@mixin reset-form-element {
+  padding: 0px;
+  background: none;
+  border: none;
+  border-radius: 0px;
+  outline: none;
+  appearance: none;
+  font-family: inherit;
+}
+
+@mixin clip-border($clip-height, $clip-width) {
+  clip-path: polygon(
+    0% 0%,
+    0% 100%,
+    100% 100%,
+    100% $clip-height,
+    calc(100% - #{$clip-width}) $clip-height,
+    calc(100% - #{$clip-width}) 0%
+  );
+}
+
+@mixin border-substitute($variant: "outlined") {
+  $border-radius: if($variant == "outlined", 4px, calc(var(--wrapper-height) / 2));
```

Comment:
Maybe use `rem` instead of `px`?

Reply:
Thanks, this file shouldn't exist anyway, since the new variant is called `field-mixins.scss` and located in another folder.